backend/src/main/java/edu/uw/cs403/plantmap/backend/controllers/ImageController.java

```diff
@@ -0,0 +1,40 @@
+package edu.uw.cs403.plantmap.backend.controllers;
+
+import edu.uw.cs403.plantmap.backend.models.Submission;
+import edu.uw.cs403.plantmap.backend.models.SubmissionServer;
+import spark.Request;
+import spark.Response;
+
+public class ImageController {
+
+    private SubmissionServer server;
+
+    public ImageController(SubmissionServer server) {
+        this.server = server;
+    }
+
+    public Object uploadImg(Request request, Response response){
+        try{
+            // parse submission id from URL
+            int sub_id = Integer.parseInt(request.params("id"));
+            server.updateSubmission(sub_id, request.bodyAsBytes());
+            response.type("text/html");
+            return "Successfully upload image";
```

Comment:
Not sure about the reply here, maybe it's best to return the `sub_id`, using response type `text/plain`?